test-plans/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+edition = "2021"
+name = "testcases"
+version = "0.1.0"
+
+# Required to avoid conflicts with parent workspace.
+[workspace]
```

Comment:
What conflicts are we talking? Can this crate not be part of our other workspace so we make sure it compiles at all times?

Reply:
@jxs did you add it here?

https://github.com/libp2p/rust-libp2p/blob/73cbbe29679f5d56d81d5477d3796e82712f9ac2/Cargo.toml#L144-L145

Reply:
Based on https://github.com/libp2p/rust-libp2p/pull/3331#discussion_r1080732134, yes I'd say let's make it part of the workspace. (Doesn't have to be in this PR)